lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -178,6 +192,11 @@ namespace Ohmbrewer {
              * Digital Pin for the temperature bus one wire protocol
              */
             int _busPin;
+
+            /**
+             * Unique ID for the temperature probe [8] char array ID code
+             */
+            char* _probeId;
```

Comment:
Why do we need a _busPin and a _probeId if we can only really intend to support one OneWire bus? It seems like it would be simpler to rename _busPin to something like _busIndex and use it to indicate where in the list of sensors the probe for the given TemperatureSensor is. Then we could use ```ow_find_sensor``` to pull a reading off that one alone. Yae/Nay?